codec/proto/protobuf.go

```diff
@@ -0,0 +1,25 @@
+package proto
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/thesyncim/exposed/codec"
+)
+
+const CodecName = "protobuf"
+
+func init() {
+	codec.RegisterCodec(ProtoCodec{})
+}
+
+type ProtoCodec struct{}
```

Comment:
type name will be used as proto.ProtoCodec by other packages, and that stutters; consider calling this Codec